properties/utils.py

```diff
@@ -146,25 +146,64 @@ class ValidationError(ValueError):
     """
 
     def __init__(self, message, reason=None, prop=None, instance=None,
-                 _error_tuples=None):
+                 _related_errors=None):
         super(ValidationError, self).__init__(message)
-        if _error_tuples is None:
-            self.error_tuples = []
+        if _related_errors is None:
+            self.related_errors = []
         else:
-            self.error_tuples = _error_tuples
+            self.related_errors = _related_errors
         if reason is not None and not isinstance(reason, string_types):
             raise TypeError('ValidationError reason must be a string')
         if prop is not None and not isinstance(prop, string_types):
             raise TypeError('ValidationError prop must be a string')
         if instance is not None and not hasattr(instance, '_error_hook'):
             raise TypeError('ValidationError instance must be a '
                             'HasProperties instance')
-        if reason or prop or instance:
-            error_tuple = ErrorTuple(message, reason, prop, instance)
-            self.error_tuples.append(error_tuple)
+        self.message = message
+        self.reason = reason
+        self.prop = prop
+        self.instance = instance
         if not getattr(instance, '_getting_validated', True):
             instance._error_hook(self.error_tuples)                           #pylint: disable=protected-access
 
+    def __str__(self, tab=1, prefix='- '):
```

Comment:
I would suggest `__unicode__` and having a `__str__` stub that returns the str version of the `__unicode__`